pkgs/servers/dns/trust-dns/default.nix

```diff
@@ -7,15 +7,15 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "trust-dns";
-  version = "0.23.0";
+  version = "0.24.0";
 
   src = fetchFromGitHub {
     owner = "bluejekyll";
     repo = "trust-dns";
```

Comment:
we should adjust the repo as well: https://github.com/hickory-dns/hickory-dns
it was renamed and is redirecting